docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.        is_cands = np.zeros(n, dtype=bool)
</pre><p>This would then be <code>np.zeros(k, dtype=bool)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I will change it.